src/main/java/eu/dissco/backend/repository/ElasticSearchRepository.java

```diff
@@ -90,9 +93,22 @@ private DigitalSpecimen mapToDigitalSpecimen(ObjectNode json) {
   }
 
   private Instant parseDate(JsonNode instantNode) {
```

Comment:
Sorry this will change in my PR... There is an item on the backlog already to fix the datatime format of elastic... driving me nuts at the moment. 
https://naturalis.atlassian.net/browse/DD-382